File: src/main/java/top/zbeboy/isy/web/bean/internship/journal/InternshipJournalBean.java
package top.zbeboy.isy.web.bean.internship.journal;

import lombok.Data;
import lombok.EqualsAndHashCode;
import top.zbeboy.isy.domain.tables.pojos.InternshipJournal;

/**
 * Created by lenovo on 2016-12-18.
 */
@EqualsAndHashCode(callSuper = false)
@Data
public class InternshipJournalBean extends InternshipJournal {
    private String createDateStr;
}

File: src/main/java/top/zbeboy/isy/web/bean/internship/journal/InternshipJournalBean.kt
package top.zbeboy.isy.web.bean.internship.journal

import top.zbeboy.isy.domain.tables.pojos.InternshipJournal

/**
 * Created by zbeboy 2017-12-29 .
 **/
class InternshipJournalBean : InternshipJournal() {
    var createDateStr: String? = null

    // 小组内统计个人日志数量
    var journalNum: Int? = 0
    var studentRealName: String? = null
    // 用于在service层取别名用
    companion object {
        @JvmField
        val JOURNAL_NUM = "journalNum"
    }
}